server/lib/db.ts

```diff
@@ -1,18 +1,22 @@
-
+import { Sequelize } from 'sequelize';
 import log from './log';
 
+const dbString = process.env.DB_STRING
+
 // Database object modeling mongoDB data
 export class Database {
   dbName: string;
+  sequelize = new Sequelize(dbString)
 
   constructor(dbName = 'postgres') {
     this.dbName = dbName;
   }
 
-  // Connect to the mongoDB server
+  // Connect to the postgres db
   async connect() {
     // connnect
-    log('Connected to MongoDB! :)');
+    await this.sequelize.authenticate();
```

Comment:
This is how we actually connect - [docs](https://sequelize.org/docs/v6/getting-started/#connecting-to-a-database)